index.html

```diff
@@ -87,6 +87,14 @@
 		text-align: center;
 	    }
 
+        #selection-region {
+        position: absolute;
+        border-style: solid;
+        border-color: #3399FF;
+        border-width: 1px;
+        background: #3399ff66;
+        display: none;
+        }
```

Comment:
Indentation? (Seems to be different than in the code around it)

Reply:
Actually indentation is a mess in this file 🤣. Many mixed space with tab. We may need another PR to fix all of them in one shot.

Reply:
Would be appreciated :-)